packages/urls.py

```diff
@@ -28,6 +28,7 @@
     path('signoffs/json/', signoff.signoffs_json, name='package-signoffs-json'),
     path('update/', views.update),
     path('sonames', views.sonames),
+    path('pkgbase-maintainer', views.pkgbase_mapping),
```

Comment:
I really dislike this endpoint name, any better ideas?

Reply:
I can totally relate why you dislike this endpoint name.

It's a hard one... I have no silver bullet... but how about simply `maintainers`? We already have like `flag` and `download` etc, thats all expected to be related to packages. so `maintainers` in that sense would simply be the endpoint for getting maintainers of packages?